patches/minecraft/net/minecraft/world/inventory/AbstractContainerMenu.java.patch

```diff
@@ -1,17 +1,32 @@
 --- a/net/minecraft/world/inventory/AbstractContainerMenu.java
 +++ b/net/minecraft/world/inventory/AbstractContainerMenu.java
-@@ -204,9 +_,11 @@
-    private void m_150407_(int p_150408_, ItemStack p_150409_, Supplier<ItemStack> p_150410_) {
-       ItemStack itemstack = this.f_38841_.get(p_150408_);
-       if (!ItemStack.m_41728_(itemstack, p_150409_)) {
-+         boolean clientStackChanged = !p_150409_.equals(itemstack, true);
-          ItemStack itemstack1 = p_150410_.get();
-          this.f_38841_.set(p_150408_, itemstack1);
+@@ -217,13 +_,15 @@
+    private void m_150435_(int p_150436_, ItemStack p_150437_, Supplier<ItemStack> p_150438_) {
+       if (!this.f_150398_) {
+          ItemStack itemstack = this.f_150394_.get(p_150436_);
+-         if (!ItemStack.m_41728_(itemstack, p_150437_)) {
++         if (!itemstack.equals(p_150437_, true)) {
+             ItemStack itemstack1 = p_150438_.get();
+             this.f_150394_.set(p_150436_, itemstack1);
+             if (this.f_150397_ != null) {
+                this.f_150397_.m_142074_(this, p_150436_, itemstack1);
+             }
+-         }
++         } else
++             if (this.f_150397_ != null)
```

Comment:
why is this `else if` on 2 lines did you mean to do an `else if` or an `else { if () {} }` ?